src/draw/DefaultDrawer.js

```diff
@@ -1592,7 +1593,15 @@ class DefaultDrawer {
       .on('click', (event, data) => {
         this.actions.linkCreation.source = this.actions.selection.current;
 
-        const componentId = this.pluginData.addComponent(data);
```

Comment:
Oh I don't see it.
That's not good, because in Terraform with don't use Component by Default but TerraformComponent, that's why  we have addComponent function in DefaultData to let plugin create it's own component.

Reply:
done